src/components/templates/PartyMap/components/Map/MapPortal.module.scss

```diff
@@ -11,15 +12,21 @@
 
 $-title-height: 42px;
 
+$-w-max-tablet: calc(#{breakpoint.$md} - 1px);
```

Comment:
[Question] Is it necessary to subtract that `1px` there? I can see that we have `min-width` a little down below, so wouldn't it mean that anything less than that would automatically mean that these `max-width` styles would be applied?

Reply:
```
breakpoint.$md: 768px;
```
so we have 2 intervals:
[...; 768) -- excluding 768 ---------- mobile ---------- max-width: 768 - 1;
and [768; ...) -- including 768 ---------- desktop ---------- min-width: 768;